src/bench/mempool_stress.cpp

```diff
@@ -56,7 +56,7 @@ static void ComplexMemPool(benchmark::Bench& bench)
         size_t n_ancestors = det_rand.randrange(10)+1;
         for (size_t ancestor = 0; ancestor < n_ancestors && !available_coins.empty(); ++ancestor){
             size_t idx = det_rand.randrange(available_coins.size());
-            Available coin = available_coins[idx];
+            Available& coin = available_coins[idx];
             uint256 hash = coin.ref->GetHash();
             // biased towards taking just one ancestor, but maybe more
             size_t n_to_take = det_rand.randrange(2) == 0 ? 1 : 1+det_rand.randrange(coin.ref->vout.size() - coin.vin_left);
```

Comment:
bench/mempool_stress.cpp:59:89: runtime error: unsigned integer overflow: 9 - 11 cannot be represented in type 'unsigned long'

Reply:
I've reverted the merge in commit 95fe477fd189ae30e76050b95280086d913e78c2 again due to this

Reply:
interesting -- why was it passing here but not on master?

Reply:
we changed from clang-12 to clang-13, maybe?